tests/ui/specs/TimePicker/TimePicker-specs.js

```diff
@@ -0,0 +1,252 @@
+					const expected = minute !== 59 ? minute + 1 : 0;
+					expect(value).to.equal(expected);
+				});
+
+				it('should not change displayed hour when decrementing the meridiem picker', function () {
+					const {hour} = extractValues(timePicker);
+					Page.spotlightRight();
+					Page.spotlightRight();
+					browser.waitUntil(() => timePicker.decrementer(timePicker.meridiem).isFocused(), {timeout: 1500,  interval: 100});
+					Page.spotlightSelect();
+
+					const {hour: value} = extractValues(timePicker);
+					expect(value).to.equal(hour);
+				});
+
+				it('should not change displayed hour when incrementing the meridiem picker', function () {
+					const {hour} = extractValues(timePicker);
+					Page.spotlightRight();
+					Page.spotlightRight();
+					Page.spotlightDown();
+					browser.waitUntil(() => timePicker.incrementer(timePicker.meridiem).isFocused(), {timeout: 1500,  interval: 100});
+					Page.spotlightSelect();
+
+					const {hour: value} = extractValues(timePicker);
+					expect(value).to.equal(hour);
+				});
+			});
+
+			describe('pointer', function () {
+				it('should select hour when opened', function () {
+					timePicker.decrementer(timePicker.hour).click();
+					browser.waitUntil(() => timePicker.decrementer(timePicker.hour).isFocused(), {timeout: 1500,  interval: 100});
+				});
+
+
+				it('should increase the hour when incrementing the picker', function () {
+					const {hour} = extractValues(timePicker);
+					timePicker.incrementer(timePicker.hour).click();
+					const {hour: value} = extractValues(timePicker);
+					const expected = hour < 12 ? hour + 1 : 1;
+					expect(value).to.equal(expected);
+				});
+
+				it('should decrease the hour when decrementing the picker', function () {
+					const {hour} = extractValues(timePicker);
+					timePicker.decrementer(timePicker.hour).click();
+					const {hour: value} = extractValues(timePicker);
+					const expected = hour > 1 ? hour - 1 : 12;
+					expect(value).to.equal(expected);
+				});
+
+				it('should increase the minute when incrementing the picker', function () {
+					const {minute} = extractValues(timePicker);
+					timePicker.incrementer(timePicker.minute).click();
+					browser.waitUntil(() => timePicker.incrementer(timePicker.minute).isFocused(), {timeout: 1500,  interval: 100});
+					const {minute: value} = extractValues(timePicker);
+					const expected = minute !== 59 ? minute + 1 : 0;
+					expect(value).to.equal(expected);
+				});
+
+				it('should decrease the minute when decrementing the picker', function () {
+					const {minute} = extractValues(timePicker);
+					timePicker.decrementer(timePicker.minute).click();
+					browser.waitUntil(() => timePicker.decrementer(timePicker.minute).isFocused(), {timeout: 1500,  interval: 100});
+					const {minute: value} = extractValues(timePicker);
+					const expected = minute !== 0 ? minute - 1 : 59;
+					expect(value).to.equal(expected);
+				});
```

Comment:
Why don't you add the below test?
```
it('should change the meridiem on hour boundaries', function () {
	const value = timePicker.timeLabel;
	// 12 hours ought to change the value text if meridiem changes
	for (let i = 12; i; i -= 1) {
		timePicker.decrementer(timePicker.hour).click();
	}
	expect(value !== timePicker.timeLabel).to.be.true();
});
```

Reply:
we don't have timeLabel in Agate. 
But, we could verifiy that the Meridiem label has changed

Reply:
Ah, I see. yes. we could.

Reply:
I added tests for this matter for 5-way and pointer